Accountancy - Add date_export on new ledger & subledger to show edit or delete link when is not export yet

aspangaro opened the pull request:
+ Repare link to create unknown account
+ Add language file

aspangaro committed:
Add language file

aspangaro committed:
Repare link to create unknown account

aspangaro committed:
Add date_export on new ledger & subledger to show edit or delete link when is not export yet

aspangaro committed:
Add language file

aspangaro committed:
Add missing language key

eldy commented on htdocs/accountancy/bookkeeping/balance.php:
```diff
@@ -321,8 +321,8 @@
 
-		if (empty($accountingaccountstatic->label) && $accountingaccountstatic->id > 0) {
-			$link = '<a href="'.DOL_URL_ROOT.'/accountancy/admin/card.php?action=update&token='.newToken().'&id='.$accountingaccountstatic->id.'">'.img_edit().'</a>';
+		if (empty($tmparrayforrootaccount['label'])) {
```
This bugs.
When you are here the accounting account already exists. Label may or may not exists.
If label does not exists and accountingaccountstatic already exists, we must link to update the account no to create.

aspangaro commented:
this functionality is not present to update an accounting account (why not) but to identify accounting accounts that do not exist in the chart of accounts probably present as a result of an accounting data import and to help with the creation.

Can we mix the two systems? Update when the account is known and creation when the account is not known.

![image](https://user-images.githubusercontent.com/2341395/103697395-2c358a00-4fa0-11eb-929e-faa230ab2ada.png)

eldy commented:
Current code manage the "update when label is not defined".
Yours is to add when no accounting exists.
So yes, best should be to manage both situation.

aspangaro committed:
Merge branch '13.0' into 13b5

aspangaro committed:
Typo

aspangaro committed:
Mix two systems for create and edit account from balance

aspangaro committed:
FIX: Accountancy - Add control on required fields when we create an account

aspangaro commented:
Before (v12), we manage create mode & in v13, we manage edit mode.

Now, when we manage the both situation : 
(For this image, I have removed the label of an existing account)

To my astonishment, I also realized that one could create an account in the chart of accounts without a number & without a label... I added a control on this point

![image](https://user-images.githubusercontent.com/2341395/103857222-59745c00-50b6-11eb-8f8d-2f8111b62827.png)
